installer/README.MD

```diff
@@ -11,11 +11,12 @@ Click the below button to clone this repository in Google Cloud and get started.
 
 Update the `variables.tfvars` file and provide the following two values. If you clicked the `Open in Cloud Shell` button this file should be opened directly in your Cloud Shell Editor.
 
-| variable              | description                                                                                               |
-| --------------------- | --------------------------------------------------------------------------------------------------------- |
-| api_key               | API Key for generating the front end connection key.                                                      |
-| project_id            | Google Cloud Project ID                                                                                   |
-| create_bq_sample_data | If set to true a BigQuery dataset and tables will be created with sample data uploaded via a Cloud Bucket |
+| variable                 | description                                                                                               |
+| ---------------------    | --------------------------------------------------------------------------------------------------------- |
+| api_key                  | API Key for generating the front end connection key.                                                      |
+| project_id               | Google Cloud Project ID                                                                                   |
+| create_bq_sample_data    | If set to true a BigQuery dataset and tables will be created with sample data uploaded via a Cloud Bucket |
+| create_tightlock_network | If set to true will create a VPC Network and Firewall rules used by the backend instance. Set it to false when deploying multiple Tightlock instances to the same cloud project. |
```

Comment:
Nit: If set to true this will create a VPC Network

Reply:
Thanks, updated it!